lib/impl/AzureRevokePresignTvm.js

```diff
@@ -46,8 +46,8 @@ class AzureRevokePresignTvm extends Tvm {
     const containerURLPublic = azureUtil.getContainerURL(accountURL, sharedKeyCredential, containerNamePublic)
 
     // revoke presign urls by resetting the default access policy
-    await azureUtil.setAccessPolicy(containerURL, params.azureStorageAccount)
-    await azureUtil.setAccessPolicy(containerURLPublic, params.azureStorageAccount, true)
+    await azureUtil.setAccessPolicy(containerURL, false)
+    await azureUtil.setAccessPolicy(containerURLPublic, true)
```

Comment:
Where is the commit that changed the signature of setAccessPolicy to take 2 args instead of 3?

Reply:
#194, got confused because that second was deadcode there since a while